src/components/Header.tsx

```diff
@@ -52,89 +54,100 @@ export function Header() {
+									</a>
+								</Link>
+							)}
+							{user && (
+								<>
+									<div className='hidden sm:ml-6 sm:block'>
+										<div className='flex items-center'>
+											<button
+												type='button'
+												className='rounded-full bg-gray-800 p-1 text-gray-400 hover:text-white focus:outline-none focus:ring-2 focus:ring-white focus:ring-offset-2 focus:ring-offset-gray-800'
+											>
+												<span className='sr-only'>View notifications</span>
+												<HiBell className='h-6 w-6' aria-hidden='true' />
+											</button>
 
-									{/* Profile dropdown */}
-									<Menu as='div' className='relative ml-3'>
-										<div>
-											<Menu.Button className='focus:underline-2 focus:underline-none flex rounded-full bg-gray-800 text-sm'>
-												<span className='sr-only'>Open user menu</span>
-												<div className='text-base font-medium text-gray-300 hover:text-white'>Tom Cook</div>
-											</Menu.Button>
-										</div>
-										<Transition
-											as={Fragment}
-											enter='transition ease-out duration-100'
-											enterFrom='transform opacity-0 scale-95'
-											enterTo='transform opacity-100 scale-100'
-											leave='transition ease-in duration-75'
-											leaveFrom='transform opacity-100 scale-100'
-											leaveTo='transform opacity-0 scale-95'
-										>
-											<Menu.Items className='absolute right-0 mt-2 w-48 origin-top-right rounded-md bg-white py-1 shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none'>
-												<Menu.Item>
-													{({ active }) => (
-														<a
-															href='#'
-															className={classNames(
-																active ? 'bg-gray-100' : '',
-																'block px-4 py-2 text-sm text-gray-700',
+											{/* Profile dropdown */}
+											<Menu as='div' className='relative ml-3'>
+												<div>
+													<Menu.Button className='focus:underline-2 focus:underline-none flex rounded-full bg-gray-800 text-sm'>
+														<span className='sr-only'>Open user menu</span>
+														<div className='text-base font-medium text-gray-300 hover:text-white'>Tom Cook</div>
+													</Menu.Button>
+												</div>
+												<Transition
+													as={Fragment}
+													enter='transition ease-out duration-100'
+													enterFrom='transform opacity-0 scale-95'
+													enterTo='transform opacity-100 scale-100'
+													leave='transition ease-in duration-75'
+													leaveFrom='transform opacity-100 scale-100'
+													leaveTo='transform opacity-0 scale-95'
+												>
+													<Menu.Items className='absolute right-0 mt-2 w-48 origin-top-right rounded-md bg-white py-1 shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none'>
+														<Menu.Item>
+															{({ active }) => (
+																<a
+																	href='#'
+																	className={classNames(
+																		active ? 'bg-gray-100' : '',
```

Comment:
```suggestion
																		active && 'bg-gray-100',
```
is this one better?

Reply:
In this case if `active` is true it'll return a string that is `bg-gray-100` but if it is false it'll return a boolean i.e. `false`. So a class named `false` will probably be added to the list of class names.

And as we're using TypeScript it'll straight away show an error saying something like the value of `boolean` is not assignable, if that makes sense :smile:  Thanks for the feedback.

Reply:
`false` will filter out by `classes.filter(Boolean).join(' ')` XD 
Thanks for the comment ❤️

Reply:
You're right. Didn't notice it before. But the TypeScript error would still be there if we use `active && 'bg-gray-100'`.